src/components/StakingVaults/AllEarnOpportunities.tsx

```diff
@@ -63,23 +67,26 @@ export const AllEarnOpportunities = () => {
       accountId: cosmosAccountId,
     },
   )
+
   const { cosmosSdkStakingOpportunities: osmosisStakingOpportunities } =
     useCosmosSdkStakingBalances({
       assetId: osmosisAssetId,
       accountId: osmosisAccountId,
     })
 
+  const lpOpportunities = useAppSelector(state => selectAggregatedEarnUserLpOpportunities(state))
```

Comment:
If you feel like stackTraces--

```suggestion
  const lpOpportunities = useAppSelector(selectAggregatedEarnUserLpOpportunities)
```